lib/erase.js

```diff
@@ -0,0 +1,34 @@
+var tessel = require('tessel')
+  , ssh = require('./tessel-ssh')
+  ;
+
+function erase(opts) {
+  var filepath = '/app';
+
+  tessel.logs.info('Connecting to remote Tessel...');
+
+  // Create a new SSH client connection object
+  ssh.createConnection(function(err, conn) {
+
+    // Throw any errors that pop up
+    if (err) throw err;
+
+    // Log that it was successful
+    tessel.logs.info('Connected.');
+
```

Comment:
This (and also tessel run / tessel push) should `/etc/init.d/tessel-app stop` to kill the old program if it exists.